api/src/main/java/org/eclipse/microprofile/openapi/annotations/links/Link.java

```diff
@@ -97,7 +97,8 @@
      * Reference value to a Link object.
      * <p>
      * This property provides a reference to an object defined elsewhere. This property and all other properties are
-     * mutually exclusive. If other properties are defined in addition to the ref property then the result is undefined.
+     * mutually exclusive. If other properties are defined in addition to the {@code ref} property then the result is
+     * undefined.
```

Comment:
`Link` may have a `description`

Reply:
I even used it in the test!

Now fixed, thanks.